src/components/AddressResolver/AddressResolver.tsx

```diff
@@ -8,29 +7,72 @@ export const AddressResolver: FunctionComponent = () => {
 
   return (
     <>
-      <div className="flex items-end">
+      <div className="flex flex-col md:flex-row md:items-end">
         <div className="flex-1 flex-col">
-          <h3>Settings: Address Resolver</h3>
-          <p className="mb-0 text-gray-500">Add third party’s endpoint to resolve addresses. </p>
+          <h3 className="font-ubuntu text-2.5 text-cloud-900">Resolver: Address</h3>
+          <p className="mb-5 text-cloud-900">Add third party’s endpoint to resolve addresses.</p>
         </div>
         <div className="flex">
           <Button
-            className="bg-tangerine text-white hover:bg-tangerine-600"
+            className="bg-white text-cerulean rounded-xl shadow-lg hover:bg-gray-50"
             onClick={() => {
               setNewEndpoint(true);
             }}
             disabled={isNewEndpoint}
           >
             <div className="flex items-center mx-0">
               <div className="col-auto mr-2">
-                <Plus />
+                <svg width="19" height="21" viewBox="0 0 19 21" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path
+                    d="M10.5712 19.2247C13.032 19.2247 14.2417 19.2247 16.7025 19.2247C16.7951 19.2211 17.255 19.2065 17.6188 18.8056C17.8522 18.5468 17.9998 18.1896 17.9998 17.7923V5.97038L13.7201 1H6.09075C5.3769 1 4.79346 1.64151 4.79346 2.42882C4.79346 5.01673 4.79346 4.62506 4.79346 7.21297"
```

Comment:
maybe let's name this svg so sth like:

```
<IconFileAdd />
```

probably consolidate/house these custom svg icons somewhere and import to here

Reply:
ok done, svg icons with 'Icon' prefix are relocated to 'src/components/UI/Icon'